.github/workflows/test.yml

```diff
@@ -0,0 +1,36 @@
+name: Test
+
+on:
+  pull_request:
+
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [2.4, 2.5, 2.6, 2.7, '3.0', 3.1, jruby-9.2]
+        os: [ubuntu-20.04, windows-2019]
+        include:
+          - { ruby: 3.1, os: ubuntu-20.04, matrix: pipeline }
+
+    runs-on: ${{ matrix.os }}
+
+    env:
+      TRAVIS_MATRIX: ${{ matrix.matrix }}
```

Comment:
What about to rename this to something more generic like `CI_MATRIX`?

Reply:
Given that there's absolutely zero reaction from repo maintainers for more than a year, I doubt it is worth working on this PR at all.

Reply:
Done!